interface/main/tabs/js/user_data_view_model.js

```diff
@@ -5,10 +5,10 @@
  * @link      http://www.open-emr.org
  * @author    Kevin Yeh <kevin.y@integralemr.com>
  * @author    Brady Miller <brady.g.miller@gmail.com>
- * @author    Ranganath Pathak <pathak@scrs1.org>
```

Comment:
You've removed Ranganath from existence :)

Reply:
lol, well we can't have that! I guess git won't give a conflict for commented code. Maybe on merge, anyway will fix and bring in.
As a note, if you haven't tried turning on User Debug while testing, I found it very helpful while doing this PR. Ya ought to try sometime.